Create OpenItemsonVisionLaunch.vb

sybethanlevy opened the pull request:
Input please re lines 12-14 and device settings; if not possible will dump the idea of course.

sybethanlevy committed:
Create OpenItemsonVisionLaunch.vb

Input please re lines 12-14 and device settings; if not possible will dump the idea of course.

Marjon80 commented on BreakpointSamples/Modifying the Standard Workflow of Vision/OpenItemsonVisionLaunch.vb:
```diff
@@ -0,0 +1,23 @@
+			If Sybiz.Vision.Platform.Security.Principal.CurrentPrincipal.IsInRole("POS") = True Then
+					
+				'Since it's POS; is it possible to 'force' KioskMode on? This seems like the way to do it, but does not work (due to Object Reference not Set...); guessing it's intentional but can't hurt to ask!
+				Dim ds As Sybiz.Vision.Platform.Admin.DeviceSettings
```
If (Sybiz.Vision.Platform.Admin.DeviceSettings.CanAccess()) Then
Dim ds As Sybiz.Vision.Platform.Admin.DeviceSettings =  Sybiz.Vision.Platform.Admin.DeviceSettings.GetObject()

Marjon80 commented on BreakpointSamples/Modifying the Standard Workflow of Vision/OpenItemsonVisionLaunch.vb:
```diff
@@ -0,0 +1,23 @@
+				'Since it's POS; is it possible to 'force' KioskMode on? This seems like the way to do it, but does not work (due to Object Reference not Set...); guessing it's intentional but can't hurt to ask!
+				Dim ds As Sybiz.Vision.Platform.Admin.DeviceSettings
+				ds.KioskMode = true
+								
```
ds = ds.Save()

sybethanlevy committed:
Update and rename OpenItemsonVisionLaunch.vb to OpenItemsOnVisionLaunch.vb

sybethanlevy commented:
Please refer above commit @Marjon80 - cheers for the pointers in the right direction! [d55d79a](https://github.com/Sybiz/Vision/pull/2/commits/d55d79a6973f6cf0cebb4ed35d6989ac5f052faf)

sybethanlevy committed:
Update OpenItemsOnVisionLaunch.vb

Resolving personal literacy problem (I forgot to put the "ds =" in despite that being in the comment!)

sybethanlevy commented:
Actually please see updated [466f30d](https://github.com/Sybiz/Vision/pull/2/commits/466f30db09a5d93861c9fec1143f8774f7b95282), now featuring one less silly mistake.
